scripts/defaultScripts.js

```diff
@@ -71,23 +73,31 @@ if (Menu.menuExists(MENU_CATEGORY) && !Menu.menuItemExists(MENU_CATEGORY, MENU_I
         isChecked: previousSetting,
     });
 }
-
 function loadSeparateDefaults() {
     var currentlyRunningScripts = ScriptDiscoveryService.getRunning();
 
     for (var i in DEFAULT_SCRIPTS_SEPARATE) {
         var shouldLoadCurrentDefaultScript = true;
+        var scriptItem = DEFAULT_SCRIPTS_SEPARATE[i];
+        if (typeof scriptItem === "object") {
+            if (previousSettingBeta) {
+                console.info("Loading Beta item " + scriptItem.beta);
```

Comment:
`console.info()` doesn't make it into the program log.
`console.log()` does, though.